part3.asm

```diff
@@ -0,0 +1,33 @@
+.data
+total:		.word	0
+max:		.word	15
+totalText:	.asciiz	"Total: "
+
+
+.text
+la $s1, max
+lw $s1, 0($s1)
+
+li $t0, 0		# loop counter
+loop:
+	addi $t0, $t0, 1	# add 1 to loop counter
+	bgt $t0, $s1, exit	# if loop counter is greater than 15 exit
+	rem $t1, $t0, 2		# find the remainder when dividing by 2 to determine if it's odd or even
```

Comment:
Works well and does what its supposed to, but why not just count by 2's instead of dividing and checking remainder?